testdata/project-v3-config/PROJECT

```diff
@@ -40,4 +40,10 @@ resources:
   webhooks:
     defaulting: true
     webhookVersion: v1
+- controller: true
+  domain: testproject.org
+  group: crew
+  kind: Laker
+  path: sigs.k8s.io/kubebuilder/testdata/project-v3-config/api/v1
```

Comment:
`Laker` didn't scaffold an api so this path does not exist, it cannot be persisted to the config file.